ui/lib/core/addon/templates/components/form-field.hbs

```diff
@@ -96,14 +96,25 @@
     label=labelString
   }}
 {{else if (eq attr.options.editType "ttl")}}
-  <TtlPicker2
-    @onChange={{action (action "setAndBroadcastTtl" valuePath)}}
-    @label={{labelString}}
-    @helperTextDisabled="Vault will use the default lease duration"
-    @helperTextEnabled="Lease will expire after"
-    @description={{attr.helpText}}
-    @initialValue={{or (get model valuePath) attr.options.setDefault}}
-  />
+  {{#if isCaCert}}
+    <TtlPicker2
+      @onChange={{action (action "setAndBroadcastTtl" valuePath)}}
+      @label={{labelString}}
+      @helperTextDisabled="Vault will use the default lease duration"
+      @helperTextEnabled="Lease will expire after"
+      @description={{attr.helpText}}
+      @initialValue='720h'
+    />
+  {{else}}
+    <TtlPicker2
+      @onChange={{action (action "setAndBroadcastTtl" valuePath)}}
+      @label={{labelString}}
+      @helperTextDisabled="Vault will use the default lease duration"
+      @helperTextEnabled="Lease will expire after"
+      @description={{attr.helpText}}
+      @initialValue={{or (get model valuePath) attr.options.setDefault}}
```

Comment:
if the only thing that `isCaCert` changes is the `initialValue`, could `initialValue` be a computed property in the component? that'd keep this template a little cleaner. whaddya think?

Reply:
heh, this idea is actually irrelevant given my other comment. i'll leave it here though just in case. :)